src/packages/storybook/withStore/withStore.js

```diff
@@ -52,9 +52,14 @@ class StoreInjector {
     }
 }
 
-const storeInjector = new StoreInjector();
+const storeInjectors = [];
 
-export { StoreInjector };
-export default function(injectedStore) {
-    return storeInjector.withStore(injectedStore);
-}
+const injectStore = injectedStore => {
+    const injector = new StoreInjector();
+    const index = storeInjectors.length;
+    storeInjectors.push(injector);
+    return storeInjectors[index].withStore(injectedStore);
```

Comment:
Actually I think to make sense to extend class `StoreInjector` with this logic and move this code to like static method `StoreInjector.create()` for example.